pyicloud/services/findmyfriends.py

```diff
@@ -0,0 +1,72 @@
+import json
+import sys
+import six
+
+from pyicloud.exceptions import PyiCloudNoDevicesException
+
+class FindMyFriendsServiceManager(object):
+    """
+    The 'Find my Friends' iCloud service
+    This connects to iCloud and returns friend data including contacts, friends,
+    following and the near-realtime latitude and longitude.
+    """
+    def __init__(self, service_root, session, params):
+        self.session = session
+        self.params = params
+        self._service_root = service_root
+        self._friend_endpoint = '%s/fmipservice/client/fmfWeb/initClient' % (
+            self._service_root,
+        )
+        self._data = {}
+
+
+    def refresh_data(self):
+        """
+        Fetches all data from Find my Friends endpoint
+        """
+        params = dict(self.params)
+        requested_data = json.dumps({
+            'clientContext': {
+                'appVersion': '1.0',
+                'contextApp': 'com.icloud.web.fmf',
+                'mapkitAvailable': True,
+                'productType': 'fmfWeb',
+                'tileServer': 'Apple',
+                'userInactivityTimeInMS': 537,
+                'windowInFocus': False,
+                'windowVisible': True
+            },
+            'dataContext': None,
+            'serverContext': None
+        })
+        req = self.session.post(
+            self._friend_endpoint,
+            data=requested_data,
+            params=params
+            )
+        self.response = req.json()
+        
+        return self.response
```

Comment:
You are not using response outside of this method.

```suggestion
        return req.json()
```

Reply:
@Quentame - It looks like the author here has intentionally copied my code almost verbatim without attribution (e.g. forking my code like a good citizen).

I would give the author the benefit of doubt he forked off my fork (or mentioned https://github.com/picklepete/pyicloud/pull/160/fileshttps://github.com/picklepete/pyicloud/pull/160/files), but the author here even used my original method docstrings.